models/DataObject/ClassDefinition/Data/Gender.php

```diff
@@ -34,8 +34,7 @@ public function configureOptions()
         $options = [
             ['key' => 'male', 'value' => 'male'],
             ['key' => 'female', 'value' => 'female'],
-            ['key' => 'other', 'value' => 'other'],
-            ['key' => '', 'value' => 'unknown'],
```

Comment:
I think we should provide a migration for `unknown` value in the db. I'm not sure though if it should be converted to `null` or `other`.